caffe2/python/pybind_state.h

```diff
@@ -40,6 +40,20 @@ void addGlobalMethods(pybind11::module& m);
 // Expose Workspace, Net, Blob
 void addObjectMethods(pybind11::module& m);
 
+/**
+ * Pybind11 would crash on py::gil_scoped_acquired if you call it from a
```

Comment:
Do we know why? Is there any documentation on this issue?

Reply:
threadstate_getframe() got wild memory access in ASAN mode. I ran it through gdb; the function was call with self=0x0. That looks suspicious to me but I'm not sure if that was the cause. I searched for this issue but nothing relevant came up.

Reply:
I didn't save the stack trace but I remember `PyEval_GetBuiltins` in the stack; I think it might be crashing in this call https://github.com/pybind/pybind11/blob/326deef2ae6c535c2b1838fd7404579b3fe497b7/include/pybind11/detail/internals.h#L142

Reply:
Can you provide a small repro that we can debug?

Reply:
@bddppq It's in this thread https://fb.facebook.com/groups/101100140348621/permalink/466378917154073/. There is stack trace there. The repro would be create an (dynamically-loaded) operator and just try to acquire the GIL. I could put up the repro on phabricator tomorrow.